skglm/penalties/separable.py

```diff
@@ -466,3 +466,48 @@ def is_penalized(self, n_features):
     def generalized_support(self, w):
         """Return a mask with non-zero coefficients."""
         return w != 0
+
+
+class NNLS(BasePenalty):
+    """Non-negative least square penalty."""
+
+    def __init__(self):
```

Comment:
to remove?

Reply:
can't be removed otherwise numba throws an error during JIT compilation